docs/explanation/selecting-populations-for-study.md

```diff
@@ -0,0 +1,29 @@
+This page is aimed at researchers working with NHS England patient record data
+as provided by the OpenSAFELY backends.
+
+## Continuity of patient data
+
+Within the NHS in England, it is usually assumed that
+a patient's current primary care record is complete:
+when a patient moves practice, their record moves with them.
+
+However, a given NHS data set may only cover a subset of patients in England:
+for example, only those included in a particular EHR vendor's database.
+Therefore, where a patient has moved to a practice outside the database,
+that patient would usually be censored from that date onwards.
+
+Specifically, note that:
+
+* From the time a patient leaves an NHS data set,
+  we cannot be confident of any facts about the patient relating to that data set,
+  unless they are completely transferred to another data set
+  available in the same backend.
+* A patient's total period of registration at one or more practices
+  comprises the entire period over which we can expect their data
+  to appear in other NHS data sets.
+  Researchers may wish to select patients with a continuous registration
+  over the time period of interest.
```

Comment:
comment: This is largely a selected quotation of words taken from a "Contracts domain modelling" document.

Reply:
Possibly! Are you worried about taking credit for others' work? If so, think about [creating a commit with multiple authors](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).

Reply:
This was a little bit of not wanting to say "I wrote this", even though I committed it.

Unfortunately, online word processors don't give you the equivalent of `git blame` but just show revision history, which makes ascribing authorship a pain. But I've done the necessary searching and it was Seb that wrote most/all of that section.